oci/table_oci_core_subnet.go

```diff
@@ -0,0 +1,295 @@
+				Description: "For an IPv6-enabled subnet, this is the IPv6 CIDR block for the subnet's private IP address space.",
+				Type:        proto.ColumnType_CIDR,
+				Transform:   transform.FromField("Ipv6CidrBlock"),
+			},
+			{
+				Name:        "ipv6_public_cidr_block",
+				Description: "For an IPv6-enabled subnet, this is the IPv6 CIDR block for the subnet's public IP address space.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Ipv6PublicCidrBlock"),
+			},
+			{
+				Name:        "ipv6_virtual_router_ip",
+				Description: "For an IPv6-enabled subnet, this is the IPv6 address of the virtual router.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Ipv6VirtualRouterIp"),
+			},
+			{
+				Name:        "prohibit_public_ip_on_vnic",
+				Description: "Indicates whether VNICs within this subnet can have public IP addresses.",
+				Type:        proto.ColumnType_BOOL,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "subnet_domain_name",
+				Description: "The subnet's domain name, which consists of the subnet's DNS label.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "virtual_router_ip",
+				Description: "The IP address of the virtual router.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "virtual_router_mac",
+				Description: "The MAC address of the virtual router.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "security_list_ids",
+				Description: "The OCIDs of the security list or lists that the subnet uses.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// tags
+			{
+				Name:        "defined_tags",
+				Description: ColumnDescriptionDefinedTags,
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "freeform_tags",
+				Description: ColumnDescriptionFreefromTags,
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Standard Steampipe columns
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(subnetTags),
+			},
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+
+			// Standard OCI columns
+			{
+				Name:        "region",
+				Description: ColumnDescriptionRegion,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "compartment_id",
+				Description: ColumnDescriptionCompartment,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CompartmentId"),
+			},
+			{
+				Name:        "tenant_id",
+				Description: ColumnDescriptionTenant,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getTenantId,
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+type subnetInfo struct {
```

Comment:
Please use the new function `ociRegionName`  for region column rather than using this custom struct.